config/routes.rb

```diff
@@ -37,7 +37,11 @@
 
   # Admin related routes
   resources :admins, only: [:index]
-  resources :reports
+  resources :reports, only: [:index, :show, :new, :edit, :update] do
```

Comment:
Do we really need all of `[:index, :show, :new, :edit, :update]`? Maybe just `[:index]` would work?